src/App.css

```diff
@@ -1,3 +1,11 @@
 .App {
   width: 100vw;
 }
+.page-container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+.content-wrap {
+  flex: 1;
+}
```

Comment:
Are you using this?

Reply:
yes, it's so that the footer stays at the bottom when the page content isn't long enough. For example, in the protected page, it would show the footer in the middle of the page.